backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -296,14 +312,32 @@ public void setOnServerErrorEventListener(OnServerErrorEventListener eventListen
     }
 
     /**
-     * Custom request handler for call to server
+     * Custom request handler for sending Backtrace reports to server
      *
      * @param requestHandler object with method which will be executed
      */
     public void setOnRequestHandler(RequestHandler requestHandler) {
         this.backtraceApi.setRequestHandler(requestHandler);
     }
 
+    /**
+     * Custom request handler for sending Backtrace unique events to server
+     *
+     * @param eventsRequestHandler object with method which will be executed
+     */
+    public void setUniqueEventsRequestHandler(EventsRequestHandler eventsRequestHandler) {
```

Comment:
Why do we want to expose this api in the public interface? Wouldn't it be better if we will set it by default in the `setOnRequestHandler` and expose an interface to override it in the backtraceMetrics implementation?

Reply:
I see other comments about moving metrics functions out of BacktraceBase and having it be a part of `BacktraceBase.Metrics.*` instead. I will try to do this